src/axelar/AxelarTransceiver.sol

```diff
@@ -0,0 +1,136 @@
+    }
+
+    function _setup(bytes calldata params) internal override {
+    }
+
+    function contractId() external pure override returns (bytes32) {
+        return keccak256('axelar-transceiver');
+    }
+
+    /**
+     * Set the bridge manager contract address
+     * @param chainId The chainId of the chain. This is used to identify the chain in the EndpointManager.
+     * @param chainName The chainName of the chain. This is used to identify the chain in the AxelarGateway.
+     */
+    function setAxelarChainId(uint16 chainId, string calldata chainName, string calldata axelarAddress) external onlyOwner {
+        AxelarTransceiverStorage storage storage_ = _storage();
+        storage_.idToAxelarChainIds[chainId] = chainName;
+        storage_.axelarChainIdToId[chainName] = chainId;
+        storage_.idToAxelarAddress[chainId] = axelarAddress;
+        storage_.axelarAddressToId[axelarAddress] = chainId;
+    }
+
+    /// @notice Fetch the delivery price for a given recipient chain transfer.
+    /// param recipientChain The Wormhole chain ID of the target chain.
+    /// param instruction An additional Instruction provided by the Transceiver to be
+    ///        executed on the recipient chain.
+    /// @return deliveryPrice The cost of delivering a message to the recipient chain,
+    ///         in this chain's native token.
+    function quoteDeliveryPrice(
+        uint16 /*recipientChain*/,
+        TransceiverStructs.TransceiverInstruction memory /*instruction*/
+    ) external view override virtual returns (uint256) {
+        // Use the gas estimation from gas service
+        return 0;
+    }
+
+    /// @dev Send a message to another chain.
+    /// @param recipientChainId The Wormhole chain ID of the recipient.
+    /// param instruction An additional Instruction provided by the Transceiver to be
+    /// executed on the recipient chain.
+    /// @param nttManagerMessage A message to be sent to the nttManager on the recipient chain.
+    function _sendMessage(
+        uint16 recipientChainId,
+        TransceiverStructs.TransceiverInstruction memory /*instruction*/,
+        bytes memory nttManagerMessage,
+        bytes32 recipientNttManagerAddress,
+        bytes32 refundAddress
+    ) internal override virtual onlyManager() {
+        emit SendTransceiverMessage(recipientChainId, nttManagerMessage, recipientNttManagerAddress, refundAddress);
+
+        AxelarTransceiverStorage storage storage_ = _storage();
+        string memory destinationContract = storage_.idToAxelarAddress[recipientChainId];
+        string memory destinationChain = storage_.idToAxelarChainIds[recipientChainId];
+
+        if(bytes(destinationChain).length == 0 || bytes(destinationContract).length == 0) revert InvalidChainId(recipientChainId);
+
+        bytes memory payload = abi.encode(manager, nttManagerMessage, recipientNttManagerAddress);
+
+        gasService.payNativeGasForContractCall{value: msg.value}(
```

Comment:
```suggestion
        gasService.payGas{value: msg.value}(
```
switch to the new [standard](https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/releases/tag/v5.7.0). on chain estimation can be passed as `false` to it as the default behaviour.

Reply:
done